srcpkgs/rmapi/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'rmapi'
 pkgname=rmapi
-version=0.0.15
+version=0.0.17
 revision=1
 build_style=go
 go_import_path=github.com/juruen/rmapi
 short_desc="CLI for accessing reMarkable table files through the cloud api"
 maintainer="Patrick Pichler <mail@patrickpichler.dev>"
-license="GPL-3.0-or-later"
+license="AGPL-3.0-only"
```

Comment:
A notice, preferably in the second line, to review `go.mod` to not use `unipdf` `3.10.0+` because of its license was changed to comercial only.

Reply:
I don't fully remember `go.mod` semantics but from what I can tell in https://github.com/juruen/rmapi/blob/a97ec2a146ef7f78b67ab671e7f88078e5c3e66a/go.mod, the version is already locked to 3.6.1 via upstream, and thus shouldn't require work from us, I think?